### ISSUE TITLE
qt5 crashes when rendering image

### ISSUE THREAD

**Issue by kmike** [scrapinghub/splash#284]
Splash crashes on OS X when running tests (Python 3, Qt 5.5.0):

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   org.qt-project.QtGui          	0x000000010a0b29c3 qt_qimageScaleAARGBA_up_xy(QImageScale::QImageScaleInfo*, unsigned int*, int, int, int, int, int, int, int, int) + 515
1   org.qt-project.QtGui          	0x000000010a0b1124 qSmoothScaleImage(QImage const&, int, int) + 804
2   org.qt-project.QtGui          	0x0000000109efb1e8 QImage::smoothScaled(int, int) const + 312
3   org.qt-project.QtGui          	0x0000000109ef54ac QImage::transformed(QTransform const&, Qt::TransformationMode) const + 2492
4   org.qt-project.QtGui          	0x0000000109ef4aab QImage::scaled(QSize const&, Qt::AspectRatioMode, Qt::TransformationMode) const + 347
5   QtGui.so                      	0x000000010a4b88d3 meth_QImage_scaled(_object*, _object*, _object*) + 467
```

full crash report: https://gist.github.com/kmike/fe45f4f715d7a30c867a

**Comment by sunu** [comment]
Has your patch to fix https://github.com/scrapinghub/splash/issues/269 landed in Qt? Should I try this on Ubuntu?

**Comment by kmike** [comment]
The patch for #269 passed code review and is in INTEGRATING stage now - see https://codereview.qt-project.org/#/c/123574/ ("Change has been successfully cherry-picked to the staging branch.").

**Comment by kmike** [comment]
Checking if you can run tests with qt5.5 on Ubuntu without craches would be great! It seems I'm getting crashes when tests are executed in parallel (py.test -n4 splash).

**Comment by kmike** [comment]
This looks like a QT issue (https://bugreports.qt.io/browse/QTBUG-48148).

**Comment by kmike** [comment]
Should be fixed in qt 5.5.1, and this is what Docker image for Splash 2.0 will use.
